src/test/debuginfo/repr-u128.rs

```diff
@@ -0,0 +1,17 @@
+// compile-flags: --emit=llvm-ir -C debuginfo=2
+#![feature(repr128)]
+
+#[repr(u128)]
+pub enum Foo {
+    Lo,
+    Hi = 1 << 64,
+}
+
+pub fn foo() -> Option<Foo> {
+    None
+}
+
+// CHECK: declare void @llvm.dbg.value
```

Comment:
Debuginfo tests are different, try looking at some of them (I don't know how to write one either, sorry, and it will be a while before I can try myself).

Reply:
I added wip commit. I'm not sure we should use GDB, LLDB or both.